cefine/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, lib, gfortran, fetchFromGitHub, makeWrapper, bash, turbomole }:
+
+let
+  binSearchPath = lib.strings.makeSearchPath "bin" [
+    bash
+    turbomole
+  ];
+
+in
+  stdenv.mkDerivation rec {
+    pname = "cefine";
+    version = "2.23";
+
+    nativeBuildInputs = [
+      gfortran
+      makeWrapper
+    ];
+
+    propagatedBuildInputs = [
+      turbomole
+    ];
+
+    src = fetchFromGitHub  {
+      owner = "grimme-lab";
+      repo = pname;
+      rev = "v${version}";
+      sha256= "0jjmvcsn5wl0djglmy0ahbx1s6d6g6f04bv2w7rrsvh9has838sa";
+    };
+
+    hardeningDisable = [
+      "format"
+    ];
+
+    dontConfigure = true;
+
+    buildPhase = ''
+      gfortran -o cefine cefine.f90
+    '';
+
+    installPhase = ''
+      mkdir -p $out/bin
+      cp cefine $out/bin/.
+    '';
+
+    postFixup = ''
+      wrapProgram $out/bin/cefine \
+        --prefix PATH : "${binSearchPath}"
+    '';
+
+    meta = with lib; {
+      description = "Non-interactive command-line wrapper around turbomoles define";
+      license = licenses.lgpl3Only;
+      homepage = "https://github.com/grimme-lab/cefine";
+      platforms = platforms.unix;
```

Comment:
You have used `platforms.unix` in other places. Is this intentional? E.g. are you using these programs also on a MAC? If not I would recommend setting it to `platforms.linux`.

Reply:
I am not using them on MAC, no. In principle I see no reason why they shouldn't work on a MAC, but I switched to Linux to be safe.

Reply:
There is no clear policy about it in upstream `nixpkgs` but I usually go with "If I can't test it on another platform than Linux than I assume it only works on Linux".